src/content/advanced/forcing-cgroupv1/index.md

```diff
@@ -0,0 +1,101 @@
+---
+linkTitle: force nodes cgroup v1
+title: Forcing nodes to use legacy Control Groups v1
+description: This article describes how to force nodes to use legacy Control Groups v1 instead the default v2.
+weight: 60
+menu:
+  main:
+    parent: advanced
+user_questions:
+ -  How can I enable cgroupsv1?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+---
+
+# Forcing cgroups v1
+
+{{< platform_support_table aws="beta=v17.0.0" azure="beta=v17.0.0" >}}
+
+## Introduction
+
+Flatcar version `3033.2.0` and above uses Control Groups v2 by default, which means all nodes will be using cgroups v2 for all kubernetes containers, as opposed to Control Groups v1 used on previous versions.
```

Comment:
```suggestion
Flatcar version `3033.2.0` and above uses control groups (cgroups) v2 by default, which means all nodes will be using cgroups v2 for all containers in Kubernetes, as opposed to cgroups v1 used on previous versions.
```

Also I would talk more about the release versions here. Which Giant Swarm release version has introduced v2, which was the last with v1? (per provider)